index.js

```diff
@@ -34,6 +39,22 @@ var cli = new Liftoff({
   completions: completion,
   extensions: interpret.jsVariants,
   v8flags: v8flags,
+  configFiles: {
+    '.gulprc': {
+      home: {
+        path: '~',
+        extensions: interpret.extensions,
+      },
+      project: {
+        path: '.',
+        extensions: interpret.extensions,
+      },
+      cwd: {
+        path: process.env.INIT_CWD,
+        extensions: interpret.extensions,
```

Comment:
Yes, I think so. For example, there were some issues which wanted to specify the path of `gulpfile.js` in a gulprc file. I suppose the cases that a directory of a config file is different from a directory of a gulpfile.js like that.